.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+#### Summary
+<!-- Provide a short summary of your changes -->
+
+#### Description
+<!-- Describe the changes in this PR here and provide some context -->
+
+#### Relevant Issues
+<!-- What are the relevant issues? Make sure there is an issue that this PR addresses here:
+ https://github.com/commercetools/commercetools-sync-java/issues and add the number(s) here please.-->
+
+#### Todo
+
+- Tests
+    - [ ] Unit 
+    - [ ] Integration
+    - [ ] Acceptance
```

Comment:
not for now: https://github.com/commercetools/commercetools-sync-java/pull/49/commits/3321f43a9f0e3871e70c16ba4a6e77ec77454d0e